include/enum_quda.h

```diff
@@ -137,23 +137,23 @@ typedef enum QudaInverterType_s {
 typedef enum QudaEigType_s {
   QUDA_EIG_TR_LANCZOS,     // Thick restarted lanczos solver
   QUDA_EIG_BLK_TR_LANCZOS, // Block Thick restarted lanczos solver
-  QUDA_EIG_IR_LANCZOS,     // Implicitly Restarted Lanczos solver (not implemented)
-  QUDA_EIG_IR_ARNOLDI,     // Implicitly Restarted Arnoldi solver (not implemented)
+  QUDA_EIG_IR_ARNOLDI,     // Implicitly Restarted Arnoldi solver
+  QUDA_EIG_BLK_IR_ARNOLDI, // Block Implicitly Restarted Arnoldi solver
   QUDA_EIG_INVALID = QUDA_INVALID_ENUM
 } QudaEigType;
 
 /** S=smallest L=largest
     R=real M=modulus I=imaniary **/
 typedef enum QudaEigSpectrumType_s {
-  QUDA_SPECTRUM_SR_EIG,
-  QUDA_SPECTRUM_LR_EIG,
-  QUDA_SPECTRUM_SM_EIG,
-  QUDA_SPECTRUM_LM_EIG,
-  QUDA_SPECTRUM_SI_EIG,
-  QUDA_SPECTRUM_LI_EIG,
+  QUDA_SPECTRUM_LM_EIG = 0,
```

Comment:
No need for the `= [...]`?

Reply:
I put them there because one can use the even/odd numbers for reversing a spectrum which is useful for simple sorting routines.